test/chains/elementChain.test.js

```diff
@@ -318,6 +318,8 @@ describe('Element chain', () => {
 			element('el-api-id').setText('text string').toString(),
 			'Setting text "text string" for "el-api-id"'
 		);
+		assert.strictEqual(elementAssert({css: 'any'}).toString(), '');
```

Comment:
I think, should be returned something similar like for not asserted command:
```javascript
suitest.element({ css: 'body' }).toString(); // Getting properties of "{"css":"body"}"
```

Reply:
Did it similar to application chain (throw invalid arguments error)